files/main.css

```diff
@@ -257,7 +257,6 @@ h1 a {
 	#panel #language {
 		font-family: 'Roboto Mono', monospace;
 		font-size: 1rem;
-		line-height: 1rem;
```

Comment:
Um, why is this line changed?

Reply:
> Um, why is this line changed?

Sorry, I didn't compare the result. Here you are.

|  Before   | After  |
|  ----  | ----  |
| <img width="226" alt="before_en" src="https://user-images.githubusercontent.com/23699926/84533162-26353e80-ad1a-11ea-8646-7e445514113b.png">  | <img width="226" alt="after_en" src="https://user-images.githubusercontent.com/23699926/84533316-7d3b1380-ad1a-11ea-80af-0792a7cdea63.png"> |
| <img width="226" alt="before_zh" src="https://user-images.githubusercontent.com/23699926/84533304-74e2d880-ad1a-11ea-9500-fd51f5052332.png">  | <img width="226" alt="after_zh" src="https://user-images.githubusercontent.com/23699926/84533333-86c47b80-ad1a-11ea-9592-6417a12c25bd.png"> |

Reply:
Ah okay, so this is required because changing from `en` to `English` requires more space.